### PR TITLE
added login-signup page

### PR DESCRIPTION
**Pull request by bishnoimukesh**
# Change Log

## Screenshot: 
Screenshot 2022-02-20 at 1 21 27 AM
Screenshot 2022-02-20 at 1 21 10 AM

## Live preview link:
https://deploy-preview-5--dreamy-thompson-b6907b.netlify.app/login.html

### CODE REVIEW

File: styles/login.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    display: flex;
+    justify-content: space-between;
+    opacity: 0.7;
+}
+
+a{
+    text-decoration: none;
+    color: var(--primary-font-color);
+    transition: 0.5s;
+}
```
Comment (sharmayatendra) [inline styles/login.css:56]: try to target class instead of element.

**Review by sharmayatendra — APPROVED** [review]
everything else looks good.

File: styles/login.css (hunk 1 above)
Comment (bishnoimukesh) [reply styles/login.css:56]: ok i will take care of it

**Review by yashpatel1797 — APPROVED** [review]
Looks good to merge.

**Pull request merged**